### ISSUE TITLE
Ability to keep remaining team members as reviewers after one team member starts a review

### ISSUE THREAD

**Issue by shssoichiro** [isaacs/github#1211]
Presently, if I add a team as a reviewer on a pull request, the first person from that team to start a review does so "on behalf of" the team, and the remaining team members no longer see that their review is requested on that pull request.

I can see that this may be useful in some cases, however in our company's case, we require at least one review from a team member, but encourage other team members to review also. Therefore, we would like a setting for all team members to remain on the PR as reviewers. I could see this being a repo-level or org-level setting.

Our workaround has been to list each individual in the CODEOWNERS files, however this is suboptimal because when a team changes, we will need to update the list of team members in multiple CODEOWNERS files, instead of in the central configuration for that team.

**Comment by shssoichiro** [comment]
From: "Ivan Žužak" <support@github.com>
> Thanks for sharing that, Joshua -- I'll pass that along to the team working on pull request reviews. 
>
> Note: you should also be aware that the repository in question is not an official GitHub repository:
>
> https://github.com/isaacs/github#issues-for-github
>
> If there's anything else you'd like to see improved -- please let us know.
>
> Best,
> Ivan

**Comment by ddebasilio** [comment]
> From: "Ivan Žužak" [support@github.com](mailto:support@github.com)
> 
> > Thanks for sharing that, Joshua -- I'll pass that along to the team working on pull request reviews.
> > Note: you should also be aware that the repository in question is not an official GitHub repository:
> > https://github.com/isaacs/github#issues-for-github
> > If there's anything else you'd like to see improved -- please let us know.
> > Best,
> > Ivan

Has there been any progress here? We face the same issue.

**Comment by excitedelektron** [comment]
Any progress on this issue? It is a really nice-to-have issue

**Comment by ddebasilio** [comment]
> Any progress on this issue? It is a really nice-to-have issue

Same here. any news?

**Comment by carbohydrates** [comment]
Hey guys, any news on this Issue?

**Comment by arturdryomov** [comment]
This works for me:

* create a team;
* open Code Review Assignment settings for the created team;
  ```
  https://github.com/orgs/{{org name}}/teams/{{team name}}/edit/review_assignment
  ```
* enable Auto Assignment, choose team members count;
* create a `.github/CODEOWNERS` file in a repo.
  ```
  * @{{org name}}/{{team name}}
  ```

Opening a pull request in the repo will auto-assign team members from the team. It is possible to skip the `CODEOWNERS` step and add the team for a review manually — GitHub will remove the team and add team members instead. However, there is a limitation in both cases — the maximum team members count which will be assigned as reviewers is `3`.

**Comment by fallXone** [comment]
I cannot believe this isn't implemented. We just switched to GitHub enterprise and this is such a pain. We add our team to every review, but one person reviewing isn't enough. We require at least 2 approvals, and as soon as I do a review, my whole team gets removed and they never see it. Even if i add the team back, it now tells me it requires my review again, even though I already reviewed it. The Enable Auto Assignment would be a great feature if you could select more than 3. I don't get why we can't use this option and select All from the dropdown, allowing a team to just auto assign everyone in the team and then remove itself.

**Comment by jebeaudet** [comment]
@fallXone I totally agree this should be implemented, it's a massive regression coming from bitbucket with the simple feature of default reviewers combined with a minimum approved reviews from default reviewers.

This being said, we found a workaround that might fit your bill (workable if your team is not too big) : 
1. Create a team `xxx-reviewers` with all the repositories that this team should review (they need to have write access in the team for this to work)
2. In this new team, create sub-teams `xxx-reviewers-1`, `xxx-reviewers-2`, etc.. The amount of sub-teams will depend on the amount of the amount of people in your team but it needs to have 3 persons MAX per sub-team. We left some extra team for new employees.
3. Enable auto assigment to each sub-team with the maximum of 3
4. In your `CODEOWNERS` file in your repositories, add this : 
```
* @your_org_name/xxx-reviewers-1 @your_org_name/xxx-reviewers-2
```

Now, on a new PR, all the members from the sub-teams will be added. You can now manage your team members within this setup, without having to change the `CODEOWNERS` file.

**Comment by BethelH** [comment]
These workarounds only work on repos without compliance restrictions that prevent large groups from being code owners, would like to comment to hopefully bump this issue ( its still sorely needed)

**Comment by ifs-anthonylecarrer** [comment]
This limitation to 3 is just so painfull ... The workaround is also painfull because i have to manage many subteams ... and what about the CODEOWNERS file when i often change it because there are turnaround in the team ... 3 years really this issue is opened ?

**Comment by Levi-Lesches** [comment]
> 3 years really this issue is opened ?

Please note that this is not [the official GitHub feedback repo](https://github.com/github/feedback/discussions/categories/general-feedback). Search there (or open a new discussion) and GitHub staff should reply shortly.

**Comment by ces-truongvu** [comment]
@ifs-anthonylecarrer Inspect the dom then change the value to 5 working for me. 

```
<input type="radio" name="team_member_count" id="team_member_count_3" value="5">
```

**Comment by nieuwsma** [comment]
Created probably a duplicate: https://github.com/github/feedback/discussions/5289 -> maybe if we upvote this one they will do something about it, since its in their official repo? 🤷🏻
